prowler/providers/aws/services/neptune/neptune_cluster_storage_encrypted/neptune_cluster_storage_encrypted.metadata.json

```
@@ -1,29 +1,41 @@
+    "Sensitive Data Identifications/Security"
+  ],
   "ServiceName": "neptune",
   "SubServiceName": "",
-  "ResourceIdTemplate": "arn:aws:rds:region:account-id:db-cluster",
+  "ResourceIdTemplate": "",
   "Severity": "high",
-  "ResourceType": "AwsRdsDbCluster",
-  "Description": "Check if Neptune Clusters storage is encrypted at rest.",
-  "Risk": "Ensure that the data available on your Amazon Neptune database instances is encrypted in order to meet regulatory requirements and prevent unauthorized users from accessing sensitive information. Encryption provides an additional layer of protection by securing your Neptune databases from unauthorized access to the underlying storage. Neptune is a fast, scalable, highly secure and fully-managed graph database service that makes it easy to build and run applications that work with deeply connected datasets.",
-  "RelatedUrl": "https://docs.aws.amazon.com/securityhub/latest/userguide/neptune-controls.html#neptune-1",
+  "ResourceType": "Other",
+  "Description": "*Neptune DB cluster* is evaluated for **encryption at rest**. The check inspects the cluster's `encrypted` flag and **passes** when storage is encrypted; it **fails** when the `encrypted` attribute is false or missing, indicating the cluster's underlying storage is not encrypted.",
+  "Risk": "**Unencrypted Neptune storage** reduces confidentiality of stored data and metadata and increases attack surface.\n\nPossible impacts:\n- Unauthorized access or data exfiltration from underlying volumes or snapshots\n- Regulatory non‑compliance\n- Greater blast radius from leaked or shared snapshots",
+  "RelatedUrl": [
+    "https://docs.aws.amazon.com/securityhub/latest/userguide/neptune-controls.html",
+    "https://docs.aws.amazon.com/securityhub/latest/userguide/neptune-controls.html#neptune-1",
+    "https://docs.datadoghq.com/security/default_rules/aws-neptune-cluster-neptune-db-clusters-should-be-encrypted-at-rest/",
+    "https://docs.prowler.com/checks/aws/general-policies/general_18/",
+    "https://docs.aws.amazon.com/neptune/latest/userguide/encrypt.html",
+    "https://docs.prowler.com/checks/aws/general-policies/general_18#terraform"
+  ],
   "Remediation": {
     "Code": {
-      "CLI": "",
-      "NativeIaC": "https://docs.prowler.com/checks/aws/general-policies/general_18#cloudformation",
-      "Other": "https://docs.prowler.com/checks/aws/general-policies/general_18/",
-      "Terraform": "https://docs.prowler.com/checks/aws/general-policies/general_18#terraform"
+      "CLI": "aws neptune create-db-cluster --db-cluster-identifier <DB_CLUSTER_ID> --engine neptune --storage-encrypted true",
+      "NativeIaC": "```yaml\nResources:\n  EncryptedNeptuneCluster:\n    Type: AWS::Neptune::DBCluster\n    Properties:\n      DBClusterIdentifier: !Sub ${DBClusterIdentifier}\n      Engine: neptune\n      StorageEncrypted: true\n```",
```

```suggestion
      "NativeIaC": "```yaml\nResources:\n  EncryptedNeptuneCluster:\n    Type: AWS::Neptune::DBCluster\n    Properties:\n      DBClusterIdentifier: !Sub ${DBClusterIdentifier}\n      StorageEncrypted: true\n```",
```